code/game/objects/items/melee/misc.dm

```diff
@@ -176,6 +176,16 @@
 		user.death(FALSE)
 	REMOVE_TRAIT(src, TRAIT_NODROP, SABRE_SUICIDE_TRAIT)
 
+/obj/item/melee/sabre/centcom
+	name = "Commander's sabre"
+	desc = "An even more elegant weapon with a purer golden grip guard, with even rarer redwood wooden grip. The blade is made of plasteel infused gold, which makes it incredibly good at cutting."
+	icon = 'modular_zubbers/icons/obj/weapons/melee/swords.dmi'
+	icon_state = "cent_sabre"
+	inhand_icon_state = "cent_sabre"
+	lefthand_file = 'modular_zubbers/icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'modular_zubbers/icons/mob/inhands/weapons/melee_righthand.dmi'
+
+
```

Comment:
Modularize this

Reply:
I've modularized most of everything, but now the outfits cannot find the paths to the sabre sheath, and the Spacepol uniform.

Reply:
Just make a new file inside modular_zubbers someplace reasonable and make sure it's ticked.

Reply:
I seemed to have fixed it, I will attempt to test it, and merge it into this.